Add note about missing web support in profiling.md (#10121)
flutter/website: src/cookbook/testing/integration/profiling.md

timeline while performing a specific task and saves a summary of the
results to a local file.

{{site.alert.note}}

>   Note that recording performance timelines is not supported on web.
>   For performance profiling on web, see
>   [Debugging performance for web apps][]

Reviewer: You don't need to say "Note" as the note macro provides that.

```suggestion
  Recording performance timelines isn't supported on web.
  For performance profiling on web, see
  [Debugging performance for web apps][]
```
